docs/docsource/Installation/configure/configuring_zenko.rst

```diff
@@ -0,0 +1,90 @@
+.. _configuring_zenko:
+
+Configuring Zenko
+=================
+
+Zenko is readily configurable using Helm to pass values set in Helm charts. 
+Helm charts are stored in Zenko/kubernetes/zenko/ and its subdirectories.
+Helm charts are YAML files with configurable values and in a Zenko deployment, 
+reconfiguration, or upgrade, Helm reads in the following order:
+
+   #. Base settings for Zenko microservices (for example, Grafana settings,
+      written to Zenko/kubernetes/zenko/charts/grafana/values.yaml).
+   #. Base settings for Zenko. These settings override the base microservice 
+      settings, and are found in Zenko/kubernetes/zenko/values.yaml.
+   #. Custom settings, which you can write to an options.yaml file. Settings
+      written to this file override settings read from the preceding
+      values.yaml file.
+
+Zenko's charts are populated by default to provide a stable, feature-rich
+deployment. It is easiest and safest to deploy Zenko using these default 
+settings in a test environment and to adjust settings on a working deployment.
+If your use case requires configuring Zenko before deployment, these
+instructions remain valid.
+
+Modifying options.yaml
+----------------------
+
+The options.yml file is not installed by default, but you added a simple one
```

Comment:
Suggestion: The options.yml file is **not present** by default, but you **can add** a simple one

Reply:
Welll...This is a reference to an earlier step. I can accept "not present," but they had to have added one (or at the very least *should* have added one) to have gotten this far. 

Either way, I need to fix this, because options.yml does not have an "a" in its suffix!